src/Http/Controllers/SwaggerViewController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace NextApps\SwaggerUi\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class SwaggerViewController
+{
+    public function __invoke(Request $request)
+    {
+        $file = collect(config('swagger-ui.files'))->filter(function ($values) use ($request) {
+            return ltrim($values['path'], '/') === $request->path();
+        })->first();
```

Comment:
```suggestion
        })->firstOrFail();
```
can you also add tests for this filtering logic

Reply:
Since the goal of this filter is to apply the correct config values to the route, isn't this already tested by the tests that check that the content of the page is modified according to the config values?